admin/tax_rates.php

```diff
@@ -152,7 +152,7 @@ function init() {
               case 'new':
                 $heading[] = array('text' => '<h4>' . TEXT_INFO_HEADING_NEW_TAX_RATE . '</h4>');
 
-                $contents = array('form' => zen_draw_form('rates', FILENAME_TAX_RATES, 'page=' . $_GET['page'] . '&action=insert', 'post', 'class="row"'));
+                $contents = array('form' => zen_draw_form('rates', FILENAME_TAX_RATES, 'page=' . $_GET['page'] . '&action=insert', 'post', 'class="form-horizontal"'));

```

Comment:
I took the `form-horizontal` out of here because it was making the text for the last field in the sidebar be right-aligned. 
Open to a better way if you have one.

Reply:
Your right, I removed it in the PR